bigbluebutton-html5/imports/ui/components/shortcut-help/useShortcutHelp.tsx

```diff
@@ -12,36 +13,41 @@ interface Accumulator {
 
 const BASE_SHORTCUTS: Array<ShortcutObject> = Meteor.settings.public.app.shortcuts;
 
-const useShortcutHelp = (param?: string) => {
+function useShortcutHelp(param?: string): Object | string | undefined {
+  const shortcut = useRef<Object | string>();
+
+  useEffect(() => {
     const ENABLED_SHORTCUTS = getFromUserSettings('bbb_shortcuts', null);
-  let shortcuts: ShortcutObject[] = Object.values(BASE_SHORTCUTS);
-
-  if (ENABLED_SHORTCUTS) {
-    shortcuts = Object.values(BASE_SHORTCUTS).map((el: ShortcutObject) => {
-      const obj = { ...el };
-      obj.descId = obj.descId.toLowerCase();
-      return obj;
-    }).filter((el: ShortcutObject) => ENABLED_SHORTCUTS.includes(el.descId.toLowerCase()));
-  }
-
-  let shortcutsString: Object | undefined = {};
-  if (param !== undefined) {
-    if (!Array.isArray(param)) {
-        shortcutsString = shortcuts
-        .filter(el => el.descId.toLowerCase() === param.toLowerCase())
-        .map(el => el.accesskey)
-        .pop();
-    } else {
-        shortcutsString = shortcuts
-        .filter(el => param.map(p => p.toLowerCase()).includes(el.descId.toLowerCase()))
-        .reduce((acc: Accumulator, current: ShortcutObject) => {
-          acc[current.descId.toLowerCase()] = current.accesskey;
-          return acc;
-        }, {});
+    let shortcuts: ShortcutObject[] = Object.values(BASE_SHORTCUTS);
+
+    if (ENABLED_SHORTCUTS) {
+      shortcuts = Object.values(BASE_SHORTCUTS).map((el: ShortcutObject) => {
+        const obj = { ...el };
+        obj.descId = obj.descId.toLowerCase();
+        return obj;
+      }).filter((el: ShortcutObject) => ENABLED_SHORTCUTS.includes(el.descId.toLowerCase()));
+    }
+
+    let shortcutsString: Object | undefined;
+    if (param !== undefined) {
+      if (!Array.isArray(param)) {
+          shortcutsString = shortcuts
+          .filter(el => el.descId.toLowerCase() === param.toLowerCase())
```

Comment:
Same case of filter in line 29
```suggestion
          .filter(el => el.descId === param.toLowerCase())
```